lib/widgets/rules_buttons.dart

```diff
@@ -64,7 +70,8 @@ class RulesChangingButtonsState extends State<RulesChangingButtons> {
                 onChanged: (value) {
                   setState(() {
                     winValue = value ?? 'Win';
```

Comment:
Make 'Win' and 'Lose' constants, so that mistake in spelling would be noticed by compiler.

Reply:
items: const ['Win', 'Lose']

Reply:
I mean different thing. I see 'Win' is repeated in the code. This is red flag, because if you change it in one place to for example 'Win!', and forget to change in others, the code will stop working.

And in this case it is better to have value boolean, not string. 'Win' should be just text to display.